shared/firestore-models/util.ts

```diff
@@ -13,15 +13,25 @@ export async function isConfiguredTargetLabel(configId: string, label: string) {
   return false;
 }
 
-/** Determine if a pull request as one of the targeted labels based on the repos config. */
-export async function hasTargetedLabel(
-  configId: string, pullRequestRef: firebase.firestore.DocumentReference) {
+/** Retrieve the target branch for a pull request based on the repos config. */
+export async function getTargetBranch(
+  configId: string, pullRequestRef: firebase.firestore.DocumentReference): Promise<string> {
     const configDoc = await getConfig(configId);
     const pullRequestDoc = await pullRequestRef.get();
+    let target = {
+      branch: '',
+      label: ''
+    }
     if (configDoc.exists && pullRequestDoc.exists) {
       const config = configDoc.data() as BranchManagerRepoConfig;
       const pullRequest = pullRequestDoc.data() as BranchManagerPullRequest;
-      return !!config.branches.find(branch => pullRequest.labels.includes(branch.label));
+      target = config.branches.find(branch => pullRequest.labels.includes(branch.label)) || target;
     }
-    return false;
+    return target.branch;
+  }
+
+/** Determine if a pull request as one of the targeted labels based on the repos config. */
+export async function hasTargetedLabel(
+  configId: string, pullRequestRef: firebase.firestore.DocumentReference): Promise<boolean> {
+    return !!(await getTargetBranch(configId, pullRequestRef));
```

Comment:
Indentation is a bit odd here. Parameters should be indented by four, and the actual code block by two 

(according to the google javascript code guidelines AFAIK)